app/_includes/components/entity_example/format/deck.md

```diff
@@ -19,7 +19,7 @@
 {% when ca_certificate %}
   The following creates a new CA Certificate:
 {% when 'plugin' %}
-  Add this section to your declarative configuration file:
+  Add this section to your [`kong.yaml`](/deck/#how-does-deck-work) configuration file:
```

Comment:
```suggestion
  Add this section to your [`kong.yaml`](/deck/get-started/) configuration file:
```
Thought this link might be more relevant since it explains how the kong.yaml file works/is created.